### PR TITLE
Simplify the structure of the app

### PR DESCRIPTION
**Pull request by gmarty**
This PR removes specific classes from the index files to only rely on the <App/> component.
This allows removing code, tests and code duplication.

### CODE REVIEW

File: index.ios.js [hunk 1]
```diff
@@ -1,27 +1,4 @@
-import React, { Component } from 'react';
-import {
-  AppRegistry,
-  StatusBar,
-  View,
-} from 'react-native';
-import { theme } from './config';
+import { AppRegistry } from 'react-native';
```
Comment (arcturus) [inline index.ios.js:1]: This file now is the same than index.android.js, should we have just a single index.js? Or the current react version will look for different entry points?

Comment (gmarty) [reply index.ios.js:1]: I tried that but it seems that these filenames are hardcoded and expected by React Native.

Comment (arcturus) [reply index.ios.js:1]: Oh, I can see that it's hardcoded in the packager :S ... so looks good for me then.

**Review by arcturus — APPROVED** [review]
shiiiip it!

**Pull request merged**